src/server.c

```diff
@@ -7308,6 +7402,12 @@ static void sigShutdownHandler(int sig) {
      * the user really wanting to quit ASAP without waiting to persist
```

Comment:
Should we update this comment?
With default configs (i.e server.shutdown_timeout=10) three CTRL+C is now needed to quit ASAP without waiting for disk writings (unless I have missed something..)
It looks like the second CTRL+C will now trigger a call to finishShutdown() which does AOF and RDB file writings.

Reply:
You're right. It's good that you highlight this. I don't know if we need this or if it's enough to just exit(1) on the second CTRL+C. The downside is that if we're waiting for replicas and exit, we don't even try to write the rdb file, but if you press CTRL+C twice, I guess you can live with data loss either way. Any strong opinions?